docker-compose.yml

```diff
@@ -147,7 +160,7 @@ services:
     networks:
       - dt-net-bridge
       - dt-net-enrollment
-    command: "bin/bundle exec rake reports:receive_and_process_reports"
+    command: "bin/bundle exec sidekiq -q assessments"
```

Comment:
should we add this to line 110 of the README

Reply:
Great catch, I will add this.